### PR TITLE
First Maven Project

### PR DESCRIPTION
**Pull request by obedjamir**
First maven project

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
```
Comment (PinakiGhosh) [inline pom.xml]: Package name and groupId doesnot match

**Review by PinakiGhosh — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (obedjamir) [reply pom.xml]: i hav renamed it to com.bankbazaar.

File: src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import com.bankbazaar.javaproblems.problem3.enums.CardRank;
+import com.bankbazaar.javaproblems.problem3.enums.CardSuite;
+
+
+/**
+ * Card Class
+ */
+public class Card implements Comparable<Card>
+{
+    public String suite;
```
Comment (PinakiGhosh) [inline src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java]: This should be enums. Not string

File: src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+     * Card constructor
+     * @param suite
+     * @param number
+     */
+    public Card(String suite, String number) throws InvalidEntryException
+    {
+        /**
+         * Validate entry
+         */
+        if (inEnum(suite, number)==false)
```
Comment (PinakiGhosh) [inline src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java]: Directly use Enum.valueOf

**Review by PinakiGhosh — CHANGES_REQUESTED** [review]

File: src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java (hunk 2 above)
Comment (obedjamir) [reply src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java]: Changed to enum type

File: src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java (hunk 3 above)
Comment (obedjamir) [reply src/main/java/com/bankbazaar/javaproblems/problem3/model/Card.java]: valueOf used for validation

**Pull request merged**